src/actions/index.js

```diff
@@ -30,3 +30,17 @@ export const fetchSyntaxes = dispatch => (
     .then(response => response.json())
     .then(json => dispatch(setSyntaxes(json)))
 );
+
+export const postSnippet = snippet => dispatch => (
+  fetch('http://api.xsnippet.org/snippets', {
+    method: 'POST',
+    mode: 'cors',
+    headers: {
+      Accept: 'application/json',
```

Comment:
Can we use quotes here? Just to be consistent with `'Content-Type'`.

Reply:
Our lint will throw an error if I will use quotes here

Reply:
🤦‍♂️

Reply:
we can add rule to always use quotes for values if this will help fill better

Reply:
sure, but not this time.